pkg/middleware/tracing.go

```diff
@@ -0,0 +1,96 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package middleware
+
+import (
+	"net/http"
+
+	"github.com/justinas/alice"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+)
+
+// NO TESTS
```

Comment:
We should start adding comments as to why we aren't adding tests for a particular module.  I'm not really disagreeing with you here, but we don't want to appear sloppy.